addons/web/static/src/views/fields/integer/integer_field.js

```diff
@@ -71,9 +90,11 @@ export const integerField = {
     extractProps: ({ attrs, options }) => ({
         formatNumber:
             options?.enable_formatting !== undefined ? Boolean(options.enable_formatting) : true,
+        humanReadable: options?.human_readable !== undefined ? Boolean(options.human_readable) : false,
```

Comment:
```suggestion
        humanReadable: Boolean(options.human_readable) 
```

Boolean(undefined) === false

Reply:
or !!options.human_readable